app/src/main/res/layout/bookmark_label_edit.xml

```diff
@@ -27,6 +27,7 @@
 			android:layout_width="0dp"
 			android:layout_height="wrap_content"
 			android:hint="@string/label_name_prompt"
+			android:inputType="textPersonName"
```

Comment:
Wouldn't maxLines property also work here?

Reply:
Didn't know there was a maxLines property. Sounds like it might. But name is actually what i wanted i think - i usually want each word to be capitalised

Reply:
I think textPersonName disables spell check. I think maxLines would probably be the right solution... if we want to set default to capitalize all words we can also use textCapWords... but I don't really like the idea of making that default. It's harder for me to "uncapitalize" first letter if keyboard makes that default than to capitalize first letter. And many of my labels I don't like all words to be capitalized...

Reply:
Fair enough. It doesn't make much difference on the search screen. I dont mind either way.

Reply:
Hmm. maxLines = 1 does not seem to prevent from entering multi-line string (it just does not expand the text input hiding other lines).